jenkins/ci.suse.de/cloud-mkphyscloud-qa-scenario-3.yaml

```diff
@@ -179,6 +179,66 @@
           timeout --signal=ALRM 150m bash -x -c ". scripts/qa_crowbarsetup.sh ; onadmin_runlist $commands;"
           ' || ret=$?
 
+          #Test nova evacuate
+          OUTPUT=$(ssh -T controller1 <<'EOF' 
+          source /root/.openrc
```

Comment:
Would be nicer to indent the whole of the `EOF` heredoc more than its surrounding `EOF` delimiters.